goon/browserassets/js/browserOutput.js

```diff
@@ -157,13 +157,10 @@ function setHighlightColor(match) {
 
 //Highlights words based on user settings
 function highlightTerms(el) {
-	var element = $(el)
-	if(!(element.mark)) { // mark.js isn't loaded; give up
-		return
-	}
 	for (var i = 0; i < opts.highlightTerms.length; i++) { //Each highlight term
 		if(opts.highlightTerms[i]) {
-			element.mark(opts.highlightTerms[i], {"element" : "span", "each" : setHighlightColor});
+			var rexp = new RegExp("("+opts.highlightTerms[i]+")","gmi")
```

Comment:
mark.js has a `markRegExp` function, is there a reason you do this instead of using that system?
https://markjs.io/#api

Reply:
Its ineffecient to get a whole library loaded into memory only to use it once for something that  can be onelined with native, non jquery code.